### DISCUSSION TITLE
First PR with GitHub Actions won't get built

### DISCUSSION THREAD

**Discussion by gtirloni** [community/community#26495] [Actions]
<p>I have a repository that doesn’t have any actions configured. So I forked it and created a PR against the master branch with a .github/workflows/main.yml file</p>
<p>It has the following contents:</p>
<p>​name: CI</p>
<p>on:<br>
push:<br>
branches: [master]<br>
pull_request:<br>
branches: [master]</p>
<p>jobs:<br>
build:</p>
<p>runs-on: ubuntu-latest</p>
<p>…</p>
<p>But the PR won’t get built by GH Actions.</p>
<p>Does Actions need the .github/workflow files to already exist? I thought anyone could create a new PR and modify the contents in that directory and the PR’s version would get used.</p>

**Comment by BrightRan** [comment, marked as answer]
<p><a class="mention" href="/gtirloni">@gtirloni</a> ,</p>
<p>By default, the workflows in forked repositories don’t run, and the workflows triggered from forked repositories also don’t run.<br>
In your case, the workaround is on the original repository, you need create a workflow runs on <strong>push</strong> event and push something to trigger this workflow. Then when you open a PR merge from the forked repository to the original repository, the workflow runs on <strong>pull_request</strong> event will be triggered normally.</p>
